Uninstalling `mongodb-community` causes CI to fail

This was introduced by #43:

```text
Run brew uninstall mongodb-community
Error: No available formula or cask with the name "mongodb-community". Did you mean mongodb-community@3.4, mongodb-community@4.4, mongodb-community@4.2, mongodb-community@4.0, mongodb-community@3.6 or mongodb-community@3.2?
Error: Process completed with exit code 1.
```

--- a/.github/workflows/macos.yaml
+++ b/.github/workflows/macos.yaml
@@ -14,9 +14,6 @@ jobs:
       shell: bash
       run: |
         echo "GITHUB_HEAD_REF=${GITHUB_HEAD_REF:-master}" >> $GITHUB_ENV
-    - name: Clean GitHub Actions environment
-      run: |
-        brew uninstall mongodb-community
     - name: Install from remote
       run: |
         echo "Installing from branch ${{ env.GITHUB_HEAD_REF }}"

